changelogs/fragments/16-ee-check.yml

```
@@ -0,0 +1,2 @@
+major_changes:
```

We generally use `minor_changes` for new features:
```suggestion
minor_changes:
```